debian/control

```diff
@@ -1,15 +1,18 @@
 Source: lutris
 Section: games
 Priority: optional
-Build-Depends: cdbs,
- debhelper,
+Build-Depends: debhelper,
+ libsdl2-2.0-0:i386,
+ libstdc++6:i386,
+ libgcc1:i386,
+ libc6:i386,
  python,
  dh-python,
  gir1.2-gtk-3.0,
  gir1.2-glib-2.0,
  python-gi,
- libgirepository1.0-dev,
-Maintainer: Mathieu Comandon <strider@strycore.com>
+ libgirepository1.0-dev
+Maintainer: Michael DeGuzis <mdeguzis@gmail.com>
```

Comment:
All the dependencies listed are included in the runtime (in both 32bit and 64bit) and they shouldn't affect the packaging process in any way.